City-of-Pori/pori-kada
# PORI-213: Additional color for pori themes

Additional color for pori themes.

To test:
1. drush fra -y && drush cc all
2. go to /admin/structure/taxonomy/theme and change the color to lightBlue
3. go to /admin/structure/menu/manage/main-menu and change the "Työ ja yrityspalvelut" menu items attribute class to theme-color-lightBlue
4. confirm that the menu color for the link is blue and the content with that theme has blue coloring
